package.json

```diff
@@ -60,29 +60,29 @@
   },
   "dependencies": {
     "@stoplight/json": "3.6.0",
-    "@stoplight/json-ref-readers": "^1.1.1",
-    "@stoplight/json-ref-resolver": "^3.0.6",
-    "@stoplight/path": "^1.3.0",
-    "@stoplight/types": "^11.2.0",
+    "@stoplight/json-ref-readers": "1.1.1",
+    "@stoplight/json-ref-resolver": "3.0.8",
+    "@stoplight/path": "1.3.1",
+    "@stoplight/types": "^11.6.0",
```

Comment:
@P0lip You've left the caret in this one. Is that on purpose?

Reply:
Yeah, this one is left on purpose. Its just typings, so should he harmless